website/docs/components/form/select/partials/accessibility/accessibility.md

```diff
@@ -1,12 +1,10 @@
-## Conformance
+## Conformance rating
 
 <Doc::Badge @type="success">Conformant</Doc::Badge>
 
 `Form::Select` is conformant when used as directed.
 
-## Accessibility
```

Comment:
In the form/primitives section, I changed this to `Accessibility Considerations` to be consistent with other pages. WDYT?

Reply:
Sounds good to me. However this work (updating the Select documentation) is [already in progress](https://hashicorp.atlassian.net/browse/HDS-1351), so I'll retire this PR in favor of the upcoming one from Heather. It will address this issue as well.